templates/openshift/common/svc/redirect.yml.j2

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: redirect
+    deployment_stamp: "{{ deployment_stamp }}"
+  name: redirect-{{ deployment_stamp }}
+  namespace: "{{ project_name }}"
+spec:
+  ports:
+{% for port in nginx_ports %}
```

Comment:
If we split nigix into app micro-service whe don't need the nginx_ports and this loop.

Reply:
Yes. I'll do this refactoring next week.

Reply:
ok